WalletWasabi.Tests/IntegrationTests/RegTests.cs

```diff
@@ -828,7 +828,7 @@ public async Task SendTestsAsync()
 				Assert.True(res2.Fee > Money.Satoshis(2 * 100)); // since there is a sanity check of 2sat/vb in the server
 				Assert.InRange(res2.FeePercentOfSent, 0, 1);
 				Assert.Single(res2.SpentCoins);
-				Assert.Equal(key2.P2wpkhScript, res2.SpentCoins.Single().ScriptPubKey);
+				Assert.Contains(new[] { key.P2wpkhScript, key2.P2wpkhScript }, x => x == res2.SpentCoins.Single().ScriptPubKey);
```

Comment:
nitpick: you can improve a bit the readability by doing:

```c#
var spentCoin = Assert.Single(res2.SpentCoins);
Assert.Contains(new[] { key.P2wpkhScript, key2.P2wpkhScript }, x => x == spentCoin.ScriptPubKey);
```

Reply:
Yes, that's much better. Done.